### ISSUE TITLE
Planning Writing Sprint

### ISSUE THREAD

**Issue by kipkurui** [BioinfoNet/hack-paper#12]
Preparation and proper sharing of responsibilities is key to getting the most out of the writing sprint. We'll use a modified version of [this](https://link.springer.com/article/10.1007/s40037-018-0442-9) to get a complete draft of the paper. 

We need:
- [ ] A collaborative platform to use
- [ ] A team leader for each aspect of the study
- [ ] Team leader to generate high-quality Figures for the paper

Steps:
- [ ] Get all the team members to decide on the best day for the Sprint
- [ ]  Agree on an achievable stretch goal for the Sprint
- [ ] Identify specific tasks (timekeeper, etc.)
- [ ] Allocate tasks in advance, and agree on what needs to be done before the Sprint

Let's continue the conversation and decide on some of the topics above.

**Comment by wanjauk** [comment]
Here are my views concerning the topics above:
1. Collaborative platforms
- **Google Docs** is the good collaborative platform for the manuscript writing. 
- We may use **Zoom** during the sprint because we are collaborating remotely. 
- We also have a **Slack** workspace that we may also use. 
- We can use **GitHub** to create milestones of the tasks we identify and for creating and resolving issues
2. The aspect of the study and team leadership. 
We had 3 aspects of the study: the status of open science in Kenya, data mining and low-cost open access publishing. We may use these respective teams, led by the team leaders, as the criteria to divide tasks among ourselves during the sprint.
3. During generation of the high quality figures, check on any pending issues on analysis. We can attend to these at this point.
4. One of the goals of the sprint is to add content to the subsections of the 3 sections of the study. We had previously identified some subsections on the [manuscript outline](https://github.com/BioinfoNet/hack-paper/blob/master/document/manuscript.md). We may use these to identify specific tasks for team members during the sprint and allocate in advance.
5. We may also:
- identify the targeted journal for the first submission
- identify more relevant literature to get us started. We already [have identified some](https://github.com/BioinfoNet/Status-of-OpenScienceKE-LiteratureSearch/blob/master/openscienceke-updatedjournal-reference/relevant_references.md) to get us started

**Comment by Shuyib** [comment]
Hi, 

I think we do the high quality figures for the paper. I think we can apply a standard theme for instance for ggplot2. We could use the theme_bw() or theme_tufte(). We can find it [ggplot2 themes](https://ggplot2-book.org/polishing.html) or we could use recommendations which can be found [Rbloggers post](https://www.r-bloggers.com/how-to-format-plots-for-publication-using-ggplot2-with-some-help-from-inkscape/). I think since majority of the plots were done with ggplot2. Careen can handle this and @Shuyib can assist.

Kamonde and Silviane could make the codebook. One to generate the codebook while the other edits the markdown in the codebook. This can be done with the [dataMaid](https://www.rdocumentation.org/packages/dataMaid/versions/1.4.0/topics/makeCodebook) package in R.

I'll export the plots and subsets of the dataframes regarding collaboration. 

Lastly, we could take some people coming Friday to make the writeup that conforms to what @kipkurui has presented above with markdown.
